app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const app = express();
+const controllers = require("./controllers/controller.topics.js");
+
+app.use(express.json());
+
+app.get("/api/topics", controllers.getTopics);
+
+app.use((err, req, res, next) => {
```

Comment:
just personal preference I think but I would have my error handling middleware below the 404 handler, functionally its the same I believe